packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -167,12 +166,12 @@ class ProcessSignal {
   @visibleForTesting
   const ProcessSignal(this._delegate);
```

Comment:
Can this take a `Platform` (from base/platform.dart) argument that defaults to a `LocalPlatform`?

Reply:
Can do - I would expect that the actual tool code should never override that

Reply:
done